force_bdss/core/data_value.py

```diff
@@ -0,0 +1,34 @@
+from traits.api import HasStrictTraits, Any, String, Enum
+
+
+class DataValue(HasStrictTraits):
```

Comment:
In the gui you will not have access to it anyway. The cli will send you a message with the relevant information. You will never have to deal with DataValue objects.